src/platforms/javascript/common/session-replay/understanding-sessions.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Understanding Sessions
+sidebar_order: 4300
+notSupported:
+  - javascript.cordova
+  - javascript.wasm
+description: 'Learn about customizing sessions with the Session Replay SDK.'
+---
+
+In most cases, setting the Replay sample rates will be all you need to do in order to capture recording data.
+For more complex cases, it is helpful to understand how sessions work and how to manually control them.
+
+## Default Session Initialization
+
+By default, Replay will immediately start a session in either `session` or `buffer` mode, depending on your `replaysSessionSampleRate` and `replaysOnErrorSampleRate`.
+
+When Replay is initialized, we check the `replaysSessionSampleRate`. If it's sampled, we start recording and sending Replay data immediately.
+
+If `replaysOnErrorSampleRate > 0`, we'll start recording in buffering mode and keep checking `replaysOnErrorSampleRate` whenever an error occurs. Once we see that it's sampled, we'll upload the Replay to Sentry and continue recording normally.
+
+If both `replaysSessionSampleRate` and `replaysOnErrorSampleRate` are `0`, the recording won't start, but you'll be able to <PlatformLink to="/session-replay/understanding-sessions/#manually-starting-recording">record manually</PlatformLink>.
+
+## Session Mode
+
+In `session` mode, Replay will continuously record and send data to Sentry. After 15 minutes of user inactivity, or a maximum duration of 60 minutes, the session will end and a new session will be initialized based on the rules from the section above.
+
+## Buffer Mode
+
+In `buffer` mode, Replay will continuously record data, but won't send any of it. It will instead keep up to the last 60 seconds in memory. When an error occurs, `replaysOnErrorSampleRate` will be checked and if it's sampled, the replay will be uploaded to Sentry and continue recording normally. After 15 minutes of user inactivity, or a maximum duration of 60 minutes, the session will end and the replay will stop.
+
+## Manually Starting Replay
+
+If both `replaysSessionSampleRate` and `replaysOnErrorSampleRate` are `0`, the recording won't start, but you'll be able to record manually:
```

Comment:
You can manually start a replay even if your Replay sample rates are greater than 0, right? If so, I wouldn't use this as the text here. We already mention this point above under "Default Session Initialization" :)

Reply:
So I changed this to:

> You can manually start a Replay session with:

plus below the code snippet:

> This can be used either if both `replaysSessionSampleRate` and `replaysOnErrorSampleRate` are `0` and thus no session has been started automatically, or if you previously stopped a session and want to start a new one (see below). `start` and `startBuffering` will throw an error if a session is currently running.